gengokit/gengokit_test.go

```diff
@@ -172,7 +168,7 @@ func TestApplyTemplateFromPath(t *testing.T) {
 		t.Fatal(err)
 	}
 
-	te, err := newTemplateExecutor(dt, goImportPath)
+	te, err := newTemplateExecutor(dt, goImportPath, goImportPath)
```

Comment:
Seeing this declaration, I'm further nudged toward the idea that passing `""` as `goPBImportPath` should be interpreted by the function as, "Just use the `goImportPath` as the protobuf import path," as long as it is documented in the function comments. Passing a zero value to allow a default value to be used is something established in the Go community. Thoughts?